ingredient/ingredient-service-bus-namespace/src/plugin.ts

```diff
@@ -1,6 +1,8 @@
 import { BaseIngredient, IngredientManager } from "@azbake/core"
 import { ARMHelper } from "@azbake/arm-helper"
 import ARMTemplate from "./arm.json"
+import { EventHubNamespaceUtils } from "@azbake/ingredient-event-hub-namespace"
```

Comment:
don't do this, use the system for loading util handlers

Reply:
Can you clarify what you mean by using the system for loading util handlers?  

One alternative to my current approach is to push the logic out to the recipe.  I used my current approach for enabling the diagnostic settings on the Event Hub Namespace itself (to transmit logs to a special diagnostics event hub).  However, I didn't want it complicate the recipes so I put it in the deployment logic.

Reply:
It dawned on me that you mean loading utils for another ingredient through IngredientManager.getIngredientFunction.  Similar to how we're loading coreutils in various ingredients.  I'll make this change.

I also reviewed my code again and realize an issue with how I'm pulling the resource group of a different ingredient.  That's what I get for testing with rgOverride.... :-)  I previously solved this in https://github.com/HomecareHomebase/azure-bake/pull/52 by modifying ingredient-utils resource_group() to accept additional params.  I can make the same change in this PR.  Or if the other PR can be reviewed again then I'll merge it into this branch?

Reply:
Add that change here so we can push this through and then look at the other PR